CODE_OF_CONDUCT.md

```diff
@@ -19,9 +19,9 @@ Examples of behavior that contributes to creating a positive environment include
 * Showing empathy towards community members and newcomers
 * Helping and mediating in cases of upsets or conflict
 
-Examples of unacceptable behavior include:
+Please think twice, before you do something like this, as it may hurt people:
 
-* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Using sexualized language or imagery and unwelcome sexual attention or advances
```

Comment:
it seems odd to say "think twice before doing unwelcome sexual attention or advances".  pretty sure you don't mean it this way but on the face of it sounds irritating.

Reply:
I had the scenario in mind that you have written something and should think twice before pressing enter; but yeah, as this can also apply to face to face encounters....